doc/API-Mastodon.md

```diff
@@ -119,17 +119,29 @@ These endpoints use the [Mastodon API entities](https://docs.joinmastodon.org/en
 
 ## Currently unimplemented endpoints
 
-These emdpoints are planned to be implemented
+These emdpoints are planned to be implemented somewhere in the future.
 
 - [`PATCH /api/v1/accounts/update_credentials`](https://docs.joinmastodon.org/methods/accounts/)
 - [`GET /api/v1/instance/activity`](https://docs.joinmastodon.org/methods/instance#weekly-activity)
 
+## Dummy endpoints
+
+These endpoints are returning empty data to avoid error messages with (for example) mobile clients.
```

Comment:
```suggestion
These endpoints are returning empty data to avoid error messages with some mobile clients.
```

Reply:
Its more "nearly all" and also it can affect desktop apps as well. So "some" isn't the right word here.